.eslintrc

```diff
@@ -12,6 +12,7 @@
     "comma-dangle": [1, "only-multiline"],
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
     "react/prefer-stateless-function": 0,
+    "react/no-did-mount-set-state": 0,
```

Comment:
This was biting us in the `jumboQuote` component since we're doing some logic on quotes in `componentDidMount`. @kylemh suggested removing the rule globally.